app/sitemap.ts

```diff
@@ -27,12 +28,19 @@ const sitemap = (): MetadataRoute.Sitemap => {
   // The current date of this request
   const currentDate = new Date().toISOString();
 
-  // This maps the URL routes into Sitemap entries
-  return [...dynamicRoutes, ...staticPaths].sort().map(route => ({
-    url: `${baseUrlAndPath}/${route}`,
-    lastModified: currentDate,
-    changeFrequency: 'always',
-  }));
+  const appRoutes = [...dynamicRoutes, ...staticPaths]
+    .sort()
+    .map(route => `${baseUrlAndPath}/${route}`);
+
+  const sitemapEntries = [...appRoutes, ...EXTERNAL_LINKS_SITEMAP].map(
```

Comment:
I don't think this one needs to be a constant, it can be a direct return.

Reply:
If I remember correctly from my typescript training, the `as metadataRoute.sitemap` checks the validity of the object at runtime to avoid errors.

Reply:
What you're saying makes no sense. There's no typescript error or issue at all by simply adding a return statement.

Also `as` doesn't check validity of anything, it is a `casting` type (cast A -> B if they're compatible)

Please simply use a return statement